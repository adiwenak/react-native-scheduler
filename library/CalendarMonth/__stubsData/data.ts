import moment from "moment"
import { Colours } from "../../shared/colour"
import { Activity, Category, CategoryActivities, CategorySelection } from "../../shared/model"
import { CategoryActivity } from "../CalendarMonth"

export const cat1: CategorySelection = {
  id: "1",
  name: "Ben Af",
  colour: Colours.Cobalt,
  isSelected: false
}

export const cat2: CategorySelection = {
  id: "2",
  name: "Cardano",
  colour: Colours.Cardinal,
  isSelected: false
}

export const cat3: CategorySelection = {
  id: "3",
  name: "Zester",
  colour: Colours.Zest,
  isSelected: false
}

export const cat4: CategorySelection = {
  id: "4",
  name: "Green Lantern",
  colour: Colours.LaPalma,
  isSelected: false
}

export const categoryCollections: Category[] = [cat1, cat2, cat3, cat4]

const date1 = moment().add(1, "days")
const date2 = moment().add(2, "days")
const date3 = moment().add(3, "days")
const date4 = moment().add(4, "days")
const date5 = moment().add(1, "weeks")
const date6 = moment().add(1, "weeks").subtract(1, "days")

export const allActivities: Activity[] = [
  {
    id: "1-1",
    category: "1",
    name: "Clean Toilet",
    startTime: date1.toDate(),
    endTime: date1.add(2, "hours").toDate()
  },
  {
    id: "1-2",
    category: "1",
    name: "Gardening",
    startTime: date2.add(1, "hours").toDate(),
    endTime: date2.add(3, "hours").toDate()
  },
  {
    id: "1-3",
    category: "1",
    name: "Gardening",
    startTime: date6.toDate(),
    endTime: date6.add(4, "hours").toDate()
  },
  {
    id: "2-1",
    category: "2",
    name: "Clean Toilet",
    startTime: date1.add(3, "hours").toDate(),
    endTime: date1.add(5, "hours").toDate()
  },
  {
    id: "2-2",
    category: "2",
    name: "Gardening",
    startTime: date2.add(2, "hours").toDate(),
    endTime: date2.add(4, "hours").toDate()
  },
  {
    id: "3-1",
    name: "Clean Toilet",
    category: "3",
    startTime: date3.add(1, "hours").toDate(),
    endTime: date3.add(5, "hours").toDate()
  },
  {
    id: "3-2",
    category: "3",
    name: "Gardening",
    startTime: date4.add(2, "hours").toDate(),
    endTime: date4.add(4, "hours").toDate()
  },
  {
    id: "4-1",
    category: "4",
    name: "BBQ",
    startTime: date1.add(4, "hours").toDate(),
    endTime: date1.add(6, "hours").toDate()
  },
  {
    id: "4-2",
    category: "4",
    name: "Clean Toilet",
    startTime: date5.add(1, "hours").toDate(),
    endTime: date5.add(5, "hours").toDate()
  },
  {
    id: "4-3",
    category: "4",
    name: "Gardening",
    startTime: date6.toDate(),
    endTime: date6.add(4, "hours").toDate()
  }
]

const map: CategoryActivities = {}

for (const activity of allActivities) {
  if (!map[activity.category]) {
    map[activity.category] = []
  }

  map[activity.category].push(activity)
}

const categoryActivities: CategoryActivity[] = []

Object.keys(map).forEach((categoryId: string) => {
  const category = categoryCollections.find((cat) => cat.id === categoryId)
  if (!category) {
    return
  }

  categoryActivities.push({
    category,
    tasks: map[categoryId]
  })
})

export { categoryActivities }
