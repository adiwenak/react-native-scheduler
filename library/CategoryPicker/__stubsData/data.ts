import { Category, CategorySelection, Colours } from "../../shared"

export const cat1: CategorySelection = {
    id: "1",
    name: "Ben Af",
    colour: Colours.Cobalt,
    isSelected: false
}

export const cat2: CategorySelection = {
    id: "2",
    name: "Cardano",
    colour: Colours.Cardinal,
    isSelected: false
}

export const cat3: CategorySelection = {
    id: "3",
    name: "Zester",
    colour: Colours.Zest,
    isSelected: false
}

export const cat4: CategorySelection = {
    id: "4",
    name: "Green Lantern",
    colour: Colours.LaPalma,
    isSelected: false
}

export const userSelectionCollection: CategorySelection[] = [
    cat1,
    cat2,
    cat3,
    cat4
]
