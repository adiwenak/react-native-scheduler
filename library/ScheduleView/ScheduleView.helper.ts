import { Activity, BaseTimeline } from "../shared"
import { GridViewItem, GroupIdentifierToColour } from "./ScheduleView.model"
import { GroupSchedule, RegroupResult } from "./ScheduleView.model"

function timeSorter(a: BaseTimeline, b: BaseTimeline) {
  return a.startTime.getTime() - b.startTime.getTime()
}

export function groupSchedule(allActivities: Activity[]): RegroupResult {
  const collections: GroupSchedule[] = []
  const sortedActivities = allActivities.sort(timeSorter)

  for (const activity of sortedActivities) {
    if (collections.length === 0) {
      const group = new GroupSchedule(activity)
      collections.push(group)
      continue
    }

    let isAdded = false
    for (const group of collections) {
      if (group.isScheduleWithinTheGroup(activity)) {
        group.addActivity(activity)
        isAdded = true
        break
      }
    }

    if (!isAdded) {
      const group = new GroupSchedule(activity)
      collections.push(group)
    }
  }

  // return regroupGroupSchedule(collections, 0)
  let longestOverlap = 0
  collections.forEach((group) => {
    group.groom()
    if (group.getNumberOfColumn() > longestOverlap) {
      longestOverlap = group.getNumberOfColumn()
    }
  })

  return {longestOverlap, groups: collections}
}

function yPosition(startTime: Date, gridStartHours: number): number {
  const toHour = startTime.getTimeInMinutes() / 60
  if (toHour > gridStartHours) {
    return toHour - gridStartHours
  } else {
    return 0
  }
}

function itemHeight(schedule: Activity): number {
  const duration = (schedule.endTime.getTimeInMinutes() - schedule.startTime.getTimeInMinutes()) / 60
  return duration
}

interface Param {
  allGroup: GroupSchedule[],
  gridMaxColumn: number,
  gridStartHour: number,
  groupToColour: GroupIdentifierToColour
}

export function mapToGridViewItem(param: Param): GridViewItem[] {
  const { allGroup, gridMaxColumn, gridStartHour, groupToColour } = param
  const items: GridViewItem[] = []

  allGroup.sort(timeSorter)

  for (const group of allGroup) {

    if (group.getNumberOfColumn() < 5) {
      const allActivities = group.flatLanes
      const columnWidth = Math.round(gridMaxColumn / group.getNumberOfColumn() * 2) / 2
      let widthRemaining = gridMaxColumn
      let counter = 0
      for (const item of allActivities) {
        // console.log(`${item.activity.name} - column span: ${item.columnSpan} - start: ${item.columnStart}`)
        const activity = item.activity
        const y = yPosition(item.activity.startTime, gridStartHour)
        const x = item.columnStart * columnWidth
        const height = itemHeight(item.activity)
        const position = { x, y }
        const widthToBeUse = columnWidth * item.columnSpan

        const dimension = { height, width: widthToBeUse }
        // console.log("position", position)
        // console.log("dimension", dimension)
        const gridItem: GridViewItem = {
          position,
          dimension,
          id: activity.id,
          name: activity.name,
          category: activity.category,
          backgroundColor: groupToColour[activity.category],
          startTime: activity.startTime,
          endTime: activity.endTime
        }

        items.push(gridItem)
        counter += 1
        widthRemaining = widthRemaining - columnWidth
      }
    } else {
      //
    }
  }

  return items
}

interface Result {
  longestOverlap: number
  items: GridViewItem[]
}

export function activityToGridViewItem(activities: Activity[], colourMap: GroupIdentifierToColour): Result {
  const group = groupSchedule(activities)
  const items = mapToGridViewItem({
    allGroup: group.groups,
    gridMaxColumn: group.longestOverlap,
    gridStartHour: 7,
    groupToColour: colourMap
  })

  return {
    longestOverlap: group.longestOverlap,
    items
  }
}
