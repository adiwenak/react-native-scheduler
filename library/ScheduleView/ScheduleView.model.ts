import { Activity, ActivityId, BaseTimeline } from "../shared"

export interface RegroupResult {
  groups: GroupSchedule[]
  longestOverlap: number
}

export interface GridViewItem extends Activity {
  position: {
    x: number
    y: number
  }
  dimension: {
    height: number
    width: number
  }
  backgroundColor: string
}

export interface GroupIdentifierToColour {
  [key: string]: string
}

interface LaneItem {
  activity: Activity
  columnSpan: number
  columnStart: number
}

function activityIsWithinActivity(activity1: Activity, activity2: Activity): boolean {
  if (activity1.startTime >= activity2.startTime && activity1.startTime < activity2.endTime) {
    return true
  } else if (activity1.endTime >= activity2.startTime && activity1.startTime < activity2.endTime) {
    return true
  } else {
    return false
  }
}

export class GroupSchedule implements BaseTimeline {
  public allItems: Activity[]
  public startTime: Date
  public endTime: Date
  public lanes?: LaneItem[][]
  public flatLanes: LaneItem[]

  constructor(schedule: Activity) {
    this.allItems = []
    this.startTime = schedule.startTime
    this.endTime = schedule.endTime
    this.flatLanes = []
    this.addActivity(schedule)
  }

  public isScheduleWithinTheGroup = (schedule: BaseTimeline): boolean => {
    if (schedule.startTime >= this.startTime && schedule.startTime <= this.endTime) {
      return true
    } else if (schedule.endTime >= this.startTime && schedule.startTime <= this.endTime) {
      return true
    } else {
      return false
    }
  }

  public getNumberOfColumn = (): number => {
    if (this.lanes) {
      return this.lanes.length
    } else {
      return 0
    }
  }

  public addGroup = (group: GroupSchedule): boolean => {
    if (!this.isScheduleWithinTheGroup(group)) {
      return false
    }

    this.allItems = [...this.allItems, ...group.allItems]

    if (group.startTime < this.startTime) {
      this.startTime = group.startTime
    }

    if (group.endTime > this.endTime) {
      this.endTime = group.endTime
    }

    return true
  }

  public groom = () => {
    if (!this.lanes) {
      return
    }

    const map: Map<ActivityId, LaneItem> = new Map()

    this.lanes.forEach((val: LaneItem[]) => {
      val.forEach((item) => {
        const existing = map.get(item.activity.id)
        if (existing) {
          existing.columnSpan = existing.columnSpan + 1
        } else {
          item.columnSpan = item.columnSpan + 1
          map.set(item.activity.id, item)
        }
      })
    })

    this.flatLanes = Array.from(map.values())
  }

  public addActivity = (schedule: Activity): boolean => {
    if (!this.isScheduleWithinTheGroup(schedule)) {
      return false
    }

    if (!this.lanes) {
      const laneItems = [
        {
          activity: schedule,
          columnStart: 0,
          columnSpan: 0
        }
      ]

      this.lanes = [laneItems]
      return true
    }

    let added = false
    this.lanes.forEach((val: LaneItem[], index: number, array: LaneItem[][]) => {
      val.forEach((item) => {
        if (!activityIsWithinActivity(schedule, item.activity)) {
          val.push({
            activity: schedule,
            columnStart: index,
            columnSpan: 0
          })
          added = true
        }
      })
    })

    if (!added) {
      const laneItems = [
        {
          activity: schedule,
          columnStart: this.lanes.length,
          columnSpan: 0
        }
      ]

      this.lanes.push(laneItems)
    }

    this.allItems.push(schedule)

    if (schedule.startTime < this.startTime) {
      this.startTime = schedule.startTime
    }

    if (schedule.endTime > this.endTime) {
      this.endTime = schedule.endTime
    }

    return true
  }
}
