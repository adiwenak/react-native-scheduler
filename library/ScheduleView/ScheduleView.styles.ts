import { StyleSheet, TextStyle, ViewStyle } from "react-native"
import { Colours } from "../shared/colour"

interface Style {
    container: ViewStyle
    containerInner: ViewStyle
    containerTimeline: ViewStyle
    containerScheduleBackground: ViewStyle
    containerSchedule: ViewStyle
    time: TextStyle,
    scheduleBox: ViewStyle
}

export const styles = StyleSheet.create<Style>({
    container: {
        flex: 1
    },
    containerInner: {
        flexDirection: "row",
        marginTop: 20,
        marginBottom: 50
    },
    containerTimeline: {
        marginTop: -13,
        borderRightWidth: 0.5,
        borderRightColor: Colours.Grey300
    },
    containerScheduleBackground: {
        flex: 1,
    },
    containerSchedule: {
        position: "absolute"
    },
    time: {
        fontSize: 12,
        textAlign: "right",
        color: Colours.Grey800,
        paddingRight: 10,
        paddingTop: 5
    },
    scheduleBox: {
        marginTop: 1,
        marginLeft: 1,
        borderRadius: 5,
    }
})
