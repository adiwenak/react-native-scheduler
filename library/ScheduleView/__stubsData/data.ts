import moment from "moment"
import { Activity } from "../../shared"
import { Colours } from "../../shared/colour"

const date1 = moment().add(1, "days").set({ hour: 0, minute: 0, second: 0, millisecond: 0 })

export const dataGroup1: Activity[] = [
  {
    id: "A1-2",
    category: "A1",
    startTime: date1.set({ hour: 13, minute: 0 }).toDate(),
    endTime: date1.set({ hour: 15, minute: 0 }).toDate(),
    name: "Test A1-2"
  },
  {
    id: "A2-1",
    category: "A2",
    startTime: date1.set({ hour: 13, minute: 30 }).toDate(),
    endTime: date1.set({ hour: 15, minute: 30 }).toDate(),
    name: "Test A2-1"
  },
  {
    id: "A1-1",
    category: "A1",
    startTime: date1.set({ hour: 11, minute: 30 }).toDate(),
    endTime: date1.set({ hour: 14, minute: 30 }).toDate(),
    name: "Test A1-1"
  },
  {
    id: "A2-2",
    category: "A2",
    startTime: date1.set({ hour: 14, minute: 0 }).toDate(),
    endTime: date1.set({ hour: 16, minute: 30 }).toDate(),
    name: "Test A2-2"
  }
]

export const dataGroup2: Activity[] = [
  {
    id: "B1-2",
    category: "B1",
    startTime: date1.set({ hour: 10, minute: 0 }).toDate(),
    endTime: date1.set({ hour: 11, minute: 0 }).toDate(),
    name: "Test B1-2"
  },
  {
    id: "B2-1",
    category: "B2",
    startTime: date1.set({ hour: 9, minute: 0 }).toDate(),
    endTime: date1.set({ hour: 11, minute: 0 }).toDate(),
    name: "Test B2-1"
  },
  {
    id: "B1-1",
    category: "B1",
    startTime: date1.set({ hour: 8, minute: 30 }).toDate(),
    endTime: date1.set({ hour: 10, minute: 30 }).toDate(),
    name: "Test B1-1"
  }
]

export const dataGroup3: Activity[] = [
  {
    id: "C1-1",
    category: "C1",
    startTime: date1.set({ hour: 11, minute: 10 }).toDate(),
    endTime: date1.set({ hour: 12, minute: 30 }).toDate(),
    name: "Test C1-1"
  },
  {
    id: "C1-2",
    category: "C1",
    startTime: date1.set({ hour: 12, minute: 20 }).toDate(),
    endTime: date1.set({ hour: 13, minute: 15 }).toDate(),
    name: "Test C1-2"
  }
]

export const dataGroup4: Activity[] = [
  {
    id: "D1-1",
    category: "D1",
    startTime: date1.set({ hour: 13, minute: 40 }).toDate(),
    endTime: date1.set({ hour: 15, minute: 0 }).toDate(),
    name: "Test D1-1"
  },
  {
    id: "D2-1",
    category: "D2",
    startTime: date1.set({ hour: 11, minute: 50 }).toDate(),
    endTime: date1.set({ hour: 14, minute: 40 }).toDate(),
    name: "Test D2"
  }
]

export const dataGroup5: Activity[] = [
  {
    id: "E1-1",
    category: "E1",
    startTime: date1.set({ hour: 15, minute: 10 }).toDate(),
    endTime: date1.set({ hour: 18, minute: 20 }).toDate(),
    name: "Test E1-1"
  },
  {
    id: "E2-1",
    category: "E2",
    startTime: date1.set({ hour: 16, minute: 35 }).toDate(),
    endTime: date1.set({ hour: 17, minute: 5 }).toDate(),
    name: "Test E2-1"
  },
  {
    id: "E2-2",
    category: "E2",
    startTime: date1.set({ hour: 14, minute: 45 }).toDate(),
    endTime: date1.set({ hour: 16, minute: 25 }).toDate(),
    name: "Test E2-2"
  }
]

export const groupBackgroundColourMap = {
  A1: Colours.Indigo,
  A2: Colours.LightBlue,
  B1: Colours.Purple,
  B2: Colours.DeepPurple,
  C1: Colours.Teal,
  C2: Colours.Lime,
  D1: Colours.Yellow,
  D2: Colours.Orange,
  E1: Colours.Cardinal,
  E2: Colours.Pink
}
