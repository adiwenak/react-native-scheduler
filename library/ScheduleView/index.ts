export { ScheduleView } from "./ScheduleView"
export * from "./ScheduleView.model"
