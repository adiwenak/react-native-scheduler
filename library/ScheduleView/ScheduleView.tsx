import * as React from "react"
import { LayoutChangeEvent, ScrollView, Text, View, ViewStyle } from "react-native"
import { Activity } from "../shared"
import { Colours } from "../shared/colour"
import { ActivityBoxView } from "./ActivityBoxView"
import { activityToGridViewItem } from "./ScheduleView.helper"
import { GridViewItem, GroupIdentifierToColour } from "./ScheduleView.model"
import { styles } from "./ScheduleView.styles"

enum ScheduleInterval {
  fifteen = 15,
  thirty = 30,
  sixty = 60
}

interface Props {
  rowHeight: number
  data: Activity[]
  colourMap: GroupIdentifierToColour
  startTime: number
  endTime: number
  interval?: ScheduleInterval
}

interface State {
  dateBoxWidth: number
  activityItems: GridViewItem[]
  numberOfColumn: number
}

export class ScheduleView extends React.PureComponent<Props, State> {
  private timeLineWidth = 60
  private scheduleContainerWidth = 0

  constructor(props: Props) {
    super(props)

    let activityItems: GridViewItem[] = []
    let numberOfColumn = 0

    if (props.data && props.data.length > 0) {
      const result = activityToGridViewItem(props.data, props.colourMap)
      numberOfColumn = result.longestOverlap
      activityItems = result.items
    }

    this.state = {
      dateBoxWidth: 0,
      activityItems,
      numberOfColumn
    }

  }

  componentDidUpdate(prevProps: Props) {
    if (this.props.data !== prevProps.data) {
      const {
        items: activityItems,
        longestOverlap: numberOfColumn } = activityToGridViewItem(this.props.data, this.props.colourMap)
      const dateBoxWidth = this.scheduleContainerWidth / numberOfColumn
      this.setState({activityItems, numberOfColumn, dateBoxWidth})
    }
  }

  render() {
    const numberOfRows = this.props.endTime - this.props.startTime + 1
    return (
      <ScrollView
        style={[styles.container]}
        onLayout={this.containerOnLayout}
        showsVerticalScrollIndicator={true}
      >
        <View style={[styles.containerInner, { height: numberOfRows * this.props.rowHeight }]}>
          <View style={[styles.containerTimeline, { width: this.timeLineWidth }]}>
            {this.renderTime()}
          </View>
          <View style={styles.containerScheduleBackground}>
            {this.renderLines()}
          </View>
          <View style={[styles.containerSchedule, { left: this.timeLineWidth }]}>
            {
              this.state.activityItems.map(this.mapItemToBox)
            }
          </View>
        </View>
      </ScrollView>
    )
  }

  private renderTime = () => {
    const startTime = this.props.startTime || 7
    const endTime = this.props.endTime || 20
    const height = this.props.rowHeight
    const allElements = []
    const style: ViewStyle = {
      height,
      width: this.timeLineWidth,
      marginRight: 15
    }
    for (let i = startTime; i <= endTime; i++) {
      const viewBox = (
        <View key={`time-${i}`} style={style}>
          <Text style={styles.time}>{`${i}:00`}</Text>
        </View>
      )

      allElements.push(viewBox)
    }

    return (
      <React.Fragment >
        {allElements}
      </React.Fragment >
    )
  }

  private renderLines = () => {
    const startTime = this.props.startTime || 7
    const endTime = this.props.endTime || 20
    const height = this.props.rowHeight
    const count = endTime - startTime
    const allElements = []
    const style: ViewStyle = {
      height,
      borderBottomWidth: 0.5,
      marginLeft: -7,
      borderColor: Colours.Grey300
    }

    const firstBoxStyle: ViewStyle = {
      ...style,
      borderTopWidth: 0.5
    }

    let firstBox = true
    for (let i = 0; i < count; i++) {
      const viewBox = (
        <View key={`line-${i}`} style={firstBox ? firstBoxStyle : style} />
      )

      allElements.push(viewBox)
      firstBox = false
    }

    return (
      <React.Fragment >
        {allElements}
      </React.Fragment >
    )
  }

  private recalculateDateBoxWidth = () => {
    const dateBoxWidth = this.scheduleContainerWidth / this.state.numberOfColumn
    this.setState({ dateBoxWidth })
  }

  private containerOnLayout = (event: LayoutChangeEvent) => {
    this.scheduleContainerWidth = event.nativeEvent.layout.width - this.timeLineWidth
    this.recalculateDateBoxWidth()
  }

  private mapItemToBox = (item: GridViewItem, idx: number) => {
    const style: ViewStyle = {
      position: "absolute",
      height: (item.dimension.height * this.props.rowHeight) - 3,
      width: (item.dimension.width * this.state.dateBoxWidth) - 1,
      top: item.position.y * this.props.rowHeight,
      left: item.position.x * this.state.dateBoxWidth,
      backgroundColor: item.backgroundColor
    }
    return (
      <ActivityBoxView
        key={`view-${idx}`}
        style={style}
        group={item.category}
        title={item.name}
      />
    )
  }
}
