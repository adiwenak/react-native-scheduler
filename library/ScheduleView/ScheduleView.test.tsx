import * as React from "react"
import { create } from "react-test-renderer"
import { dataGroup1, dataGroup2, dataGroup3, groupBackgroundColourMap } from "./__stubsData/data"
import { ScheduleView } from "./ScheduleView"

const allActivity = [...dataGroup1, ...dataGroup2, ...dataGroup3]

describe("<ScheduleView />", () => {
  it("should render with given props", () => {
    const component = create(
      <ScheduleView
        data={allActivity}
        rowHeight={70}
        startTime={7}
        endTime={23}
        colourMap={groupBackgroundColourMap}
      />
    ).toJSON()

    expect(component).toMatchSnapshot()
  })
})
