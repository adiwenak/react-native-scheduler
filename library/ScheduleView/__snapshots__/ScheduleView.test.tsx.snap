// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ScheduleView /> should render with given props 1`] = `null`;
